bin/addons/godot_whisper/capture_stream_to_text.gd

```diff
@@ -140,7 +140,15 @@ func _on_timer_timeout():
 	if is_running:
 		_speech_to_text_singleton.add_audio_buffer(buffer)
 
-func _remove_special_characters(message: String):
+func _remove_special_characters(message: String, is_partial: bool):
+	if is_partial == false:
```

Comment:
is this needed? This function should remove all such things with the for below it.

Reply:
If ` is_ Partial==false ` and does not contain the [end] character:

This indicates that the text appears in the following state: because there is too much data in pcmf32, pcmf32 has already deleted the first segment of audio. Therefore, the corresponding first segment of text is in the completed state, but the generated text in the second segment is actually in the incomplete state,

So I will use it as a segmentation criterion to determine whether two signals need to be emitted, which helps maintain the coherence of the generated text display